queue/driver_mongodb.go

```diff
@@ -395,30 +395,42 @@ func (d *mongoDB) Next(ctx context.Context) amboy.Job {
 		job    amboy.Job
 	)
 
-	qd = bson.M{
-		"$or": []bson.M{
-			{
-				"status.completed": false,
-				"status.in_prog":   false,
-			},
-			{
-				"status.completed": false,
-				"status.in_prog":   true,
-				"status.mod_ts":    bson.M{"$lte": time.Now().Add(-lockTimeout)},
-			},
-		},
-	}
-
-	if d.respectWaitUntil {
+	if d.useNewQuery {
```

Comment:
What's this change?

Reply:
I think this was wrong in earlier versions, and we cleaned up the default.